sources/api/migration/migrations/v1.14.0/kubelet-config-settings/src/main.rs

```diff
@@ -5,6 +5,14 @@ use std::process;
 fn run() -> Result<()> {
     migrate(AddSettingsMigration(&[
         "settings.kubernetes.cpu-manager-policy-options",
+        "settings.kubernetes.cpu-cfs-quota-enforced",
+        "settings.kubernetes.shutdown-grace-period",
+        "settings.kubernetes.shutdown-grace-period-for-critical-pods",
+        "settings.kubernetes.eviction-soft",
+        "settings.kubernetes.eviction-soft-grace-period",
+        "settings.kubernetes.eviction-max-pod-grace-period",
+        "settings.kubernetes.memory-manager-policy",
+        "settings.kubernetes.memory-manager-reserved-memory",
```

Comment:
This one needs to go in a different `AddPrefixes` migration since we allow user-defined keys underneath it.